knimin/handlers/ag_pulldown.py

```diff
@@ -55,10 +63,19 @@ def post(self):
             blanks = self.get_argument('blanks').split(',')
         else:
             blanks = []
-        if self.get_argument('external'):
+
+        # query which surveys have been selected by the user
+        if self.get_argument('selected_ag_surveys', []):
```

Comment:
I figured that it can be even easier than that, if we use get_aguments (not the s). Then, you can provide a list of strings as parameters.